### ISSUE TITLE
Confusing reader view results from GitHub Pull Request

### ISSUE THREAD

**Issue by pdehaan** [mozilla/readability#199]
### Steps to reproduce:
1. Go to https://github.com/mozilla/fxa-oauth-server/pull/251
2. Switch to reader view mode.

#### Before:
![db_changes_in_preparation_for_third-party_reliers_by_rfk_ _pull_request__251_ _mozilla_fxa-oauth-server_ _github_and_mopad__fxa-web-coordination_and_pocket__terms_of_service](https://cloud.githubusercontent.com/assets/557895/7418611/e74a3612-ef23-11e4-9ae6-1d00324212ba.png)

#### After:
![db_changes_in_preparation_for_third-party_reliers_by_rfk_ _pull_request__251_ _mozilla_fxa-oauth-server](https://cloud.githubusercontent.com/assets/557895/7418623/0f535198-ef24-11e4-9657-ccce4fdc5d23.png)

---

If I understand correctly, readability is pulling the text from a seemingly random file (in this case test/api.js in the pull request, which is 7/8ths of the way down the page) and using that as the preferred text, and ignoring the actual pull request comment text in the Conversation tab:

![db_changes_in_preparation_for_third-party_reliers_by_rfk_ _pull_request__251_ _mozilla_fxa-oauth-server_ _github](https://cloud.githubusercontent.com/assets/557895/7418682/8878d872-ef24-11e4-88a0-e410bcc265fc.png)

Not hugely surprising I guess, unless we do have some special logic for GitHub pages. It looks like if I save that page to Pocket and try and read it, it doesn't offer a reader view at all and just takes me back to the original GitHub page.

Similarly, Safari doesn't offer a reader view version of the page.

Embed.ly was able to parse the page a bit better, getting the page title and description (from the pull request comment, as expected), but doesn't seem to show the full contents of the GitHub comment (the `content` response is `null`): http://embed.ly/docs/explore/extract?url=https%3A%2F%2Fgithub.com%2Fmozilla%2Ffxa-oauth-server%2Fpull%2F251

**Comment by gijsk** [comment]
Yeah, I'm afraid we might need blocklists for URLs at least for the initial release, to block twitter, github and potentially other sites. Not happy about it, but I don't really see another way.

**Comment by pdehaan** [comment]
Even worse, I don't think we want to block all of twitter.com/*, only certain listing pages. I think for static URLs to specific tweets, we should support a reader view.

**Comment by gijsk** [comment]
Maybe we should, but I don't think that's feasible for 38.0.5.
